docs/learning/SupportedPlatforms.md

```diff
@@ -16,11 +16,8 @@ They should run on most Windows, Linux and MacOS versions with [Tier 1 support](
 | - | - |
 | Node 14.x | ✔️ |
 | Node 12.x | ✔️ |
-| Node 10.x* | ✔️ |
-| Node <10.17 | ❌ |
-
-> \* Node 10.x is no longer in LTS and thus is not routinely tested.
-
+| Node <12.20 | ❌ |

```

Comment:
I'm worried saying `12.x ✔️` when `<12.20 ❌` 

How about:

| Node Version | Supported |
| - | - |
| Node 12 (>12.19) | ✔️ |
| Node <12.20 | ❌

Reply:
That makes sense. I'll update it

Reply:
Updated but used `(>=12.20)` instead of `(>12.19)` as I think it made more sense when taking a quick look at the graph.

Reply:
@wgoehrig Just updated the docs again to reflect the change to track Node versions due to our N-API dependency. Let me know what you think of the wording.